build/common.project.props

```diff
@@ -15,7 +15,9 @@
     <NETCoreTargetFramework>netcoreapp2.1</NETCoreTargetFramework>
     <NetStandardVersion>netstandard2.0</NetStandardVersion>
     <TargetFrameworksExe>$(NETFXTargetFramework);$(NETCoreTargetFramework)</TargetFrameworksExe>
+    <TargetFrameworksExe Condition="'$(DotNetBuildFromSource)' == 'true'">$(NETCoreTargetFramework)</TargetFrameworksExe>
```

Comment:
Please use IsBuildOnlyXPLATProjects instead. 
It was added in order to not leak source-build logic throughout our build.
 
https://github.com/NuGet/NuGet.Client/blob/502505da868f79b1c4fc1704dae6642138adb30c/build/common.project.props#L45